keras_cv/losses/iou_loss_test.py

```diff
@@ -54,26 +54,6 @@ def test_output_shape_relative(self):
             [0.2, 0.1, 0.3, 0.3, 7, 0.48],
         ]
 
-        iou_loss = IoULoss(bounding_box_format="rel_xyxy")
+        iou_loss = IoULoss(bounding_box_format="rel_xyxy", axis=-1)
 
         self.assertAllEqual(iou_loss(y_true, y_pred).shape, ())
-
-    def test_output_value(self):
```

Comment:
why removing numerical test?

Reply:
The IoU loss is not pairwise anymore. I can write a new numerical test for this.